projects/go-lib/src/lib/components/go-action-sheet/go-panel/go-panel.component.spec.ts

```diff
@@ -44,22 +44,31 @@ describe('GoPanelComponent', () => {
       expect(component.action.emit).toHaveBeenCalled();
     });
 
-    it('should close the action sheet', () => {
-      parentComponent.showContent = true;
+    it('should call panelClicked', () => {
+      spyOn(component, 'panelClicked');
 
       component.handleAction();
 
-      expect(parentComponent.showContent).toBeFalsy();
+      expect(component.panelClicked).toHaveBeenCalled();
     });
   });
 
-  describe('closeActionSheet', () => {
-    it('should set parent action sheet\'s showContent to false', () => {
+  describe('panelClicked', () => {
+    it('should set parent action sheet\'s showContent to false if closeOnClick is true', () => {
       parentComponent.showContent = true;
 
-      component.closeActionSheet();
+      component.panelClicked();
 
       expect(parentComponent.showContent).toBeFalsy();
     });
```

Comment:
Should we specifically set `component.closeOnClick = true` here since it doesn't look like were doing any clean up after these tests which means if at any point the order of these tests got changed this test would fail?